src/web/themes/custom/bcgov_career/src/sass/_components.scss

```diff
@@ -2502,69 +2502,69 @@ span.question-prefix {
             }
 
             .compare-career {
-                display: block;
-                position: relative;
-                padding-left: 35px;
-                margin-bottom: 12px;
-                cursor: pointer;
-                font-size: 22px;
-                -webkit-user-select: none;
-                -moz-user-select: none;
-                -ms-user-select: none;
-                user-select: none;
-
-                input {
-                    position: absolute;
-                    opacity: 0;
-                    cursor: pointer;
-                    height: 18px;
-                    width: 18px;
-                    left: 0;
-                    z-index: 1;
-
-                    ~ .checkmark {
-                        background-color: #fff;
-                        border: 1px solid #797979;
-                    }
-
-                    &:checked ~ .checkmark {
-                        background-color: #fff;
-                        border: 1px solid #234075;
-                    }
-
-                    &:checked ~ .checkmark:after {
-                        display: block;
-                    }
-
-                    &:focus-visible{
-                        ~ .checkmark {
-                            outline: 2px dashed #d21c22;
-                        }
-                    }
-                }
-
-                .checkmark {
-                    position: absolute;
-                    top: 0;
-                    left: 0;
-                    height: 20px;
-                    width: 20px;
-                    background-color: #eee;
-
-                    &:after {
-                        content: "";
-                        position: absolute;
-                        display: none;
-                        left: 6px;
-                        top: 1px;
-                        width: 7px;
-                        height: 13px;
-                        border: solid #234075;
-                        border-width: 0 3px 3px 0;
-                        -webkit-transform: rotate(45deg);
-                        -ms-transform: rotate(45deg);
-                        transform: rotate(45deg);
-                    }
+              display: flex;
+              justify-content: center;
+              position: relative;
+
+              margin-bottom: 12px;
+              cursor: pointer;
+              font-size: 22px;
+              -webkit-user-select: none;
+              -moz-user-select: none;
+              -ms-user-select: none;
+              user-select: none;
+
+              input {
+                  position: absolute;
+                  opacity: 0;
+                  cursor: pointer;
```

Comment:
For some reason, previous authors set the input opacity to "0" and then added a bootstrap checkbox as separate HTML.